src/com/TTT/GameOptionsUI.java

```diff
@@ -0,0 +1,43 @@
+package com.TTT;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Scanner;
+
+public class GameOptionsUI {
```

Comment:
In hindsight, GameOptions doesn't make sense as a class name and I was thinking ahead too much.